modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -0,0 +1,150 @@
+:_content-type: CONCEPT
+[id="installation-nutanix-installer-infra-reqs_{context}"]
+= Environment requirements
+
+Before you install an {product-title} cluste, review the following Nutanix AOS environment requirements.
+
+[id="installation-nutanix-installer-infra-reqs-account_{context}"]
+== Required account privileges
+
+The installation program requires access to an account with privileges to read and create the required resources. Using an account that has global administrative privileges is the simplest way to access all of the necessary permissions.
+
```

Comment:
Is "global administrative privileges" the correct term to describe a Prism Central admin account that has the necessary level of privileges to install a cluster. If restrictions within an organization prevent a user account from have global privileges, what subset of privileges are required.

Example of this doc: https://docs.openshift.com/container-platform/4.10/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned

Reply:
@rvanderp3 To clarify - I am interpreting your comment as that the doc simply state that the account that is used to install the cluster requires administrative privileges and that the subsequent tables that detail `Roles and privileges required for installation` and `Required permissions and propagation settings` should be deleted. Is that correct?

Reply:
@mjpytlak yes, that is correct.

Reply:
Done.